nnvm/python/nnvm/top/nn.py

```diff
@@ -167,16 +167,22 @@ def compute_contrib_conv2d_NCHWc(attrs, inputs, _):
     padding = attrs.get_int_tuple("padding")
     strides = attrs.get_int_tuple("strides")
     dilation = attrs.get_int_tuple("dilation")
+    channels = attrs.get_int("channels")
```

Comment:
Just for curiosity. Are channels here "out channels? If yes, it will better to name it appropriately for clarity

Reply:
Yes, will do.